templates/changeVobLocation/gamesave.d

```diff
@@ -4,7 +4,7 @@
 
 // ### TODO: Replace original position of {0, 0, 0} ###
 // ### TODO: Replace corrected position of {0, 0, 0} ###
-// ### TODO: Replace callbacks of of Hlp_Is_WhateverYouSeek ###
+// ### TODO: Replace callbacks of Hlp_Is_Todo ###
```

Comment:
I'd like to move this block of to-dos rather inline to where the developer has to make adjustments. That clarifies the location of the changes, one doesn't have to adjust to-dos in two places, and thanks to Botwyn's comments the locations will be nicely highlighted.

Example:
```D
const float G1CP_{ISSUE_NUM_PAD}_{SHORTNAME}_OriginalPos[3] = {0, 0, 0}; // ### TODO Add coordinates, e.g. from spacer
```

Botwyn can handle the "TODO" with and with a following colon. I would opt for leaving it out, to reduce the overall line length in the templates.